### ISSUE TITLE
/github help should offer more detailed subscribe information

### ISSUE THREAD

**Issue by zachleat** [integrations/slack#718]
This is super helpful information and should be elevated to be included with the `/github help` command: https://github.com/integrations/slack#configuration

Specifically the features available to subscribe and the subscribe defaults!

**Comment by zachleat** [comment]
More info at: https://twitter.com/zachleat/status/1052284815406493697

**Comment by carchrae** [comment]
i agree - no way to see what it is subscribed to unless you add a new subscription to something.  

a lame workaround is to subscribe to commits again, 

```
/github subscribe <YOUR_ORG>/<YOUR_REPO> commits
```

it is very tedious to add all the subscriptions too.  i'd rather enable all of them then disable ones which are too noisy.

**Comment by scepticulous** [comment]
> i agree - and there is no way to see what it is subscribed to unless you add a new subscription to something.

Did you try `/github subscribe list features`?

**Comment by scepticulous** [comment]
Hi @zachleat 👋, 
thanks for raising this topic. We are always happy to hear feedback and get suggestions for things that improve the daily use of this integration.

> This is super helpful information and should be elevated to be included with the `/github help` command: https://github.com/integrations/slack#configuration
> 
> Specifically the features available to subscribe and the subscribe defaults!

This issue is already quite old. Sorry, you had to wait. The list of available features is now included in the `help` command. Does this satisfy your request? If so, feel free to close this out.
